opencti-platform/opencti-front/src/private/components/cases/case_incidents/Root.tsx

```diff
@@ -126,148 +125,148 @@ const RootCaseIncidentComponent = ({ queryRef, caseId }) => {
   const isOverview = location.pathname === `/dashboard/cases/incidents/${caseData?.id}`;
   const paddingRight = getPaddingRight(location.pathname, caseData?.id, '/dashboard/cases/incidents', false);
   const { isFeatureEnable } = useHelper();
-  const canManageAuthorizedMembers = caseData?.currentUserAccessRight === 'admin' && isFeatureEnable('CONTAINERS_AUTHORIZED_MEMBERS');
+
+  if (!caseData) {
+    return <ErrorNotFound />;
+  }
+
+  const currentAccessRight = useGetCurrentUserAccessRight(caseData);
```

Comment:
useGetCurrentUserAccessRight should be called on caseData.currentUserAccessRight. I don't understand how it can work with caseData

Reply:
Yep, don't think this is working